packages/material-ui/src/TableBody/tableBodyClasses.d.ts

```diff
@@ -0,0 +1,7 @@
+import { TableBodyClassKey } from './TableBody';
+
+declare const tableBodyClasses: Record<TableBodyClassKey, string>;
```

Comment:
I'm not sure that it will be the way forward, at least, we haven't yet decided with https://github.com/mui-org/material-ui/pull/24736#issuecomment-772888279.

Reply:
Ah apologizes, I've missed that conversation, I thought the PR was merged. In my opinion, it makes sense that we use the `props.classes` the most, as far as I remember that was the first proposal in the PR. I don't mind if we use the `ComponentClassesKeys`, but we will have the pseudo keys there as well, which may be misleading, as the `ComponentClassesKeys` are used for the `stylesOverrides` in the theme. I'd say we shouldn't block the open migration PRs because of this anyway, as we will anyway need to update the classes in all components once we decide on the best option we will have.